libfaketime/__init__.py

```diff
@@ -103,17 +104,22 @@ def __init__(self, datetime_spec, only_main_thread=True):
             _datetime = dateutil.parser.parse(datetime_spec)
 
         self.time_to_freeze = _datetime  # freezegun compatibility
-        self.libfaketime_spec = _datetime.strftime('%Y-%m-%d %T %f')
 
     def _should_fake(self):
         return self.only_main_thread and threading.current_thread().name == 'MainThread'
 
+    def _format_datetime(self, _datetime):
+        return _datetime.strftime('%Y-%m-%d %T %f')
+
+    def tick(self, delta=datetime.timedelta(seconds=1)):
+        self.time_to_freeze += delta
+        os.environ['FAKETIME'] = self._format_datetime(self.time_to_freeze)
```

Comment:
Hmm, I am not a freezegun expert, so I decided to take a better look on it's code and see how it behaves in that case. What I found out is that freezegun has two different "ticks" and they are completely unrelated to each other.

The first tick that they mention is the [tick argument](https://github.com/spulec/freezegun#tick-argument). When you set it to `True`, instead of freezing the time, it resets the time to a certain period but will keep it running normally after that. This "tick" attribute is not related to this PR.

The second "tick" is what they call [manual ticks](https://github.com/spulec/freezegun#manual-ticks) and it's just a way to change the time from inside an already open context manager. This is what this PR tries to implement

So about our question, unlike libfaketime, the instance returned by the freezegun __enter__ is a [different class](https://github.com/spulec/freezegun/blob/master/freezegun/api.py#L341) which means that when you execute a manual tick like this:

```python
with freeze_time(initial_datetime) as frozen_datetime:
    frozen_datetime.tick()
```

You are actually accessing the tick method from the [FrozenDateTimeFactory](https://github.com/spulec/freezegun/blob/master/freezegun/api.py#L257) class.

That also means that if try access the tick attribute without a context manager open, what you are actually going to see is a [undocumented boolean](https://github.com/spulec/freezegun/blob/master/freezegun/api.py#L277) that represents the value of tick argument. Therefore, the following code would raise an exception in freezegun:

```python
>> freeze_time('2011-01-01').tick()
TypeError: 'bool' object is not callable
```

Since directly accessing the tick attribute in freezegun is an undocumented behaviour,  I personally see no harm in having these small differences between both libraries, but if you prefer I could update this PR so that the libfaketime class behaves more like the freezegun class.